src/passes/CoalesceLocals.cpp

```diff
@@ -438,7 +438,7 @@ void CoalesceLocals::applyIndices(std::vector<Index>& indices, Expression* root)
   }
   // update type list
   auto numParams = getFunction()->getNumParams();
-  Index newNumLocals = 0;
+  Index::value_type newNumLocals = 0;
```

Comment:
if we need this here, don't we need it every single place that we define `Index i` for a loop etc.? That sounds bad :(